images/Dockerfile

```diff
@@ -33,6 +33,9 @@ COPY --from=build /build/dist/flanneld /opt/bin/flanneld
 COPY dist/mk-docker-opts.sh /opt/bin/
 COPY --from=build /iptables-wrapper/iptables-wrapper-installer.sh /
 COPY --from=build /iptables-wrapper/bin/iptables-wrapper /
+# check manually that iptables-legacy and iptables-nft are present since
+# iptables-wrapper-installer.sh sanity check doesn't work for multi-arch build
```

Comment:
but if you look at the last line, it says `--no-sanity-check`, are we really doing the sanity check?

Reply:
no because it doesn't work.
Instead I use `which` to check that `iptables-nft` and `iptables-legacy` exist in the image